### PR TITLE
docs: Reuse VCN

### PR DESCRIPTION
**Pull request by karthicgit**
Doc updated for reuse VCN.

### CODE REVIEW

File: docs/reusevcn.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+:uri-rel-file-base: link:{uri-repo}/blob/main
+:uri-rel-tree-base: link:{uri-repo}/tree/main
+
+:uri-docs-reuse-vcn: {uri-rel-file-base}/docs/terraformoptions.adoc#reuse-vcn
+
+== Assumptions
+Internet Gateway, NAT Gateway,Service Gateway was created in the VCN and route table preexist.
+
+== Instructions
+By default the module will create VCN ,gateways etc and then create Oracle kubernetes engine cluster. There are usecases where enduser might want to reuse their existing VCN . To do that set `create_vcn = false` and pass other required values mentioned in {uri-docs-tfoption}[terraformoption.adoc] under Reuse VCN section.
```
Comment (hyder) [inline docs/reusevcn.adoc:19]: The link is incorrect

File: docs/reusevcn.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+You can pass either the vcn_display_name or vcn_id .If you have duplicate VCN display names recommend to use vcn_id. If both values are passed vcn_id will take precedence. Similarly for Internet Gateway and NAT Gateway route table you can either pass displayname(`ig_route_table_display_name` and `nat_route_table_display_name`) or pass the id (`ig_route_table_id` and `nat_route_table_id`) .
```
Comment (hyder) [inline docs/reusevcn.adoc:21]: Indicate what is required in the route tables.

File: docs/terraformoptions.adoc [hunk 3]
```diff
@@ -250,6 +250,51 @@ EOT
...
+|Description
+|Values
+|Default
+
+|create_vcn
+|Whether to create the VCN.
+|true/false
+|true
+
+|vcn_display_name
```
Comment (hyder) [inline docs/terraformoptions.adoc:268]: display_name is not unique. We shouldn't use this.

**Review by hyder — CHANGES_REQUESTED** [review]
Please review links, assumptions and input variables

File: docs/terraformoptions.adoc (hunk 3 above)
Comment (karthicgit) [reply docs/terraformoptions.adoc:268]: This option is provided for usecase in devops where they dont have to update for different environment.

Comment (karthicgit) [reply docs/terraformoptions.adoc:268]: display name is kept for now .

**Review by hyder — APPROVED** [review]
Approved

**Comment by agregory999** [comment]
Typo in the Reuse VCN section - display name

**Pull request merged**